.github/workflows/ci.yml

```diff
@@ -49,6 +49,30 @@ jobs:
       - name: Test
         run: ./ci/${{ matrix.crates }}.sh
 
+  # Test crates on non x86_64 architectures.
+  cross:
+    strategy:
+      matrix:
+        crates:
+          - crossbeam
```

Comment:
I wonder if we can maintain only one set of crates for both x86-64 and the cross-target platforms.  Maybe later we will add more crates, and I'd wish we should change the CI script in only one place...

Reply:
Agreed. A single 'crates' set is certainly preferable. (It makes a matrix a little more complicated, but it should be possible.)

Reply:
> It makes a matrix a little more complicated, but it should be possible.

Hmm, the way I thought would work didn't really work. I'll look for other ways.

Reply:
I addressed this by testing all crates at once in cross tests. (07a1366. The way of adjusting the matrix did not work.)

This should make non-x86-64 platform test time a bit longer than x86-64 platform, at this time. However, eventually, x86-64 platform test will take more time because sanitizers (#591), miri (#578), etc. will be added to x86-64 platform test. So I think this is fine.

Reply:
Thank you for the revised PR. I just want to suggest another (potential) alternative: can we also test x86-64 platform in cross?  If so, we can uniformly treat all platforms using cross.

Reply:
cross [doesn't provide images for windows-msvc and darwin](https://github.com/rust-embedded/cross#supported-targets), so I don't think using cross on those platforms will no benefit.
However, I'll take a look for a way to merge the `test` and `cross` jobs into one job.